mw-webapp/src/component/icon/Icon.cy.tsx

```diff
@@ -0,0 +1,54 @@
+import {Icon, IconDictionary, IconSize} from "src/component/icon/Icon";
+import {getDataCy} from "src/utils/cyTesting/getDataCy";
+
+const ICON_CY = "icon";
+const ICON_RENDER = "EyeOpenedIcon";
```

Comment:
why render? it looks like just icon name

Reply:
Done